Microsoft.Toolkit.Uwp/Helpers/SystemInformation.cs

```diff
@@ -15,69 +15,73 @@
 using Windows.Security.ExchangeActiveSyncProvisioning;
 using Windows.System;
 using Windows.System.Profile;
+using Windows.System.UserProfile;
 
 namespace Microsoft.Toolkit.Uwp.Helpers
 {
     /// <summary>
     /// Defines class providing information of OS and application
     /// </summary>
-    public class SystemInformation
+    public static class SystemInformation
     {
         /// <summary>
         /// Gets Application's name
         /// </summary>
-        public string ApplicationName { get; private set; }
+        public static string ApplicationName { get; }
 
         /// <summary>
         /// Gets Application's version
         /// </summary>
-        public PackageVersion ApplicationVersion { get; private set; }
+        public static PackageVersion ApplicationVersion { get;  }
 
         /// <summary>
         /// Gets current culture
         /// </summary>
-        public string Culture { get; private set; }
+        public static string Culture { get; }
```

Comment:
And actually, given Culture is a really standard .net thing, do we really need this property at all?